PlayCover/Views/PlayCoverApp.swift

```diff
@@ -26,6 +26,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         if ProcessInfo.processInfo.isLowPowerModeEnabled {
             powerModal()
         }
+        URLCache.iconCache.removeAllCachedResponses()
```

Comment:
Do the icon caches need to be removed on launch?

Reply:
Was asked since is the old cache method and need to be cleared since no more necessary